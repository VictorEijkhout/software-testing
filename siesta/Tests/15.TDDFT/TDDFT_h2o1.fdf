SystemName TDDFT_h2o
SystemLabel TDDFT_h2o


################################################################

xc.functional LDA
xc.authors CA
Atom.Ignore.Ghosts

MeshCutoff 150.0 Ry

SolutionMethod diagon
ElectronicTemperature 5 meV

DM.MixingWeight 0.2
DM.NumberPulay 3
DM.UseSaveDM F

################################################################

TDED.WF.Initialize T

%block ExternalElectricField
0.200  -0.200  0.000  V/Ang
%endblock ExternalElectricField

###############################################################
NumberOfAtoms 3
NumberOfSpecies 2

%block ChemicalSpeciesLabel
1 8 O
2 1 H
%endblock ChemicalSpeciesLabel

PAO.BasisSize DZP

LatticeConstant 10.0 Ang

%block LatticeVectors
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0
%endblock LatticeVectors

AtomicCoordinatesFormat NotScaledCartesianAng

%block AtomicCoordinatesAndAtomicSpecies
0.401406    0.437428    0.768445 1
0.004485   -0.004402    0.000174 2
0.004730   -0.004345    1.536873 2
%endblock AtomicCoordinatesAndAtomicSpecies
