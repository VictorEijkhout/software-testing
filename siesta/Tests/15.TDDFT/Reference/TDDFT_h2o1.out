Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/15.TDDFT/work/TDDFT_h2o1
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:40:41

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.35144
************************** Dump of input data file ****************************
SystemName TDDFT_h2o
SystemLabel TDDFT_h2o
################################################################
xc.functional LDA
xc.authors CA
Atom.Ignore.Ghosts
MeshCutoff 150.0 Ry
SolutionMethod diagon
ElectronicTemperature 5 meV
DM.MixingWeight 0.2
DM.NumberPulay 3
################################################################
TDED.WF.Initialize T
%block ExternalElectricField
0.200  -0.200  0.000  V/Ang
%endblock ExternalElectricField
###############################################################
NumberOfAtoms 3
NumberOfSpecies 2
%block ChemicalSpeciesLabel
1 8 O
2 1 H
%endblock ChemicalSpeciesLabel
PAO.BasisSize DZP
LatticeConstant 10.0 Ang
%block LatticeVectors
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0
%endblock LatticeVectors
AtomicCoordinatesFormat NotScaledCartesianAng
%block AtomicCoordinatesAndAtomicSpecies
0.401406    0.437428    0.768445 1
0.004485   -0.004402    0.000174 2
0.004730   -0.004345    1.536873 2
%endblock AtomicCoordinatesAndAtomicSpecies
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: TDDFT_h2o
reinit: -----------------------------------------------------------------------
reinit: System Label: TDDFT_h2o
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    8 Label: O
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: O
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/O.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf

---- Pseudopotential check for O 

Pseudized shells:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
Valence configuration for ps generation: (assumed as above)

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  15.999        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for O                     (Z =   8)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1366
V l=1 = -2*Zval/r beyond r=  1.1332
V l=2 = -2*Zval/r beyond r=  1.1298
V l=3 = -2*Zval/r beyond r=  1.1264
All V_l potentials equal beyond r=  1.1355
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1366
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.667 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     76.727 Ry
atom: Maximum radius for chlocal:    1.37241
atom: Maximum radius for r*vlocal+2*Zval:    1.25469

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.293209   el= -1.742414   Ekb=  9.076872   kbcos=  0.325320
GHOST: No ghost state for L =  1
   l= 1   rc=  1.284119   el= -0.676589   Ekb= -8.194485   kbcos= -0.396010
GHOST: No ghost state for L =  2
   l= 2   rc=  1.433152   el=  0.013307   Ekb= -2.071665   kbcos= -0.003690
GHOST: No ghost state for L =  3
   l= 3   rc=  1.540891   el=  0.019584   Ekb= -0.817137   kbcos= -0.000371

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    3.560943
                 energy =   -1.731321
                kinetic =    1.587329
    potential(screened) =   -3.318651
       potential(ionic) =  -11.260325

   izeta = 2
                 rmatch =    2.319910
              splitnorm =    0.150000
                 energy =   -1.319217
                kinetic =    2.736746
    potential(screened) =   -4.055963
       potential(ionic) =  -12.745464

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.290924
                 energy =   -0.665565
                kinetic =    4.934810
    potential(screened) =   -5.600376
       potential(ionic) =  -13.346429

   izeta = 2
                 rmatch =    2.782200
              splitnorm =    0.150000
                 energy =   -0.468052
                kinetic =    6.936835
    potential(screened) =   -7.404886
       potential(ionic) =  -15.957492

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    4.290924
                 energy =    2.236783
                kinetic =    4.278617
    potential(screened) =   -2.041834
       potential(ionic) =   -8.171412
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   4.290924

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=2  polorb=1  (1s)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=0  polorb=0  (2p)  (perturbative polarization orbital) (from 1s)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.285354
                 energy =   -0.456120
                kinetic =    0.900445
    potential(screened) =   -1.356565
       potential(ionic) =   -1.890080

   izeta = 2
                 rmatch =    3.582433
              splitnorm =    0.150000
                 energy =   -0.289774
                kinetic =    1.649095
    potential(screened) =   -1.938868
       potential(ionic) =   -2.516427

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s
Split based on tail norm

   izeta = 1
                     rc =    5.285354
                 energy =    0.642323
                kinetic =    1.282472
    potential(screened) =   -0.640149
       potential(ionic) =   -1.117275
atom: Total number of Sankey-type orbitals:  5

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.285354

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    8 O                       # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
O                     2                    # Species label, number of l-shells
 n=2   0   2                         # n, l, Nzeta 
   3.561      2.320   
   1.000      1.000   
 n=2   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   4.291      2.782   
   1.000      1.000   
H                     1                    # Species label, number of l-shells
 n=1   0   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.285      3.582   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file O.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.75855   0.82662   1.45215  1        1
siesta:      0.00848  -0.00832   0.00033  2        2
siesta:      0.00894  -0.00821   2.90427  2        3

siesta: System type = molecule  

initatomlists: Number of atoms, orbitals, and projectors:      3    23    34

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   150.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =    58.0226 K
redata: Fix the spin of the system                  =   F
redata: Write Initial TDWF                          =   T
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 3
mix.SCF:    Linear mixing weight                 =     0.200000
mix.SCF:    Mixing weight                        =     0.200000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2000
  weight.linear 0.2000
  history 3
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
       10.000000    0.000000    0.000000
        0.000000   10.000000    0.000000
        0.000000    0.000000   10.000000

outcell: Cell vector modules (Ang)   :   10.000000   10.000000   10.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :   1000.0000
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=23 nrows=23 sparsity=1.0000 nnzs=529, refcount: 7>
  <dData1D:(new from dSpData1D) n=529, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=23 nrows=23 sparsity=1.0000 nnzs=529, refcount: 8>
  <dData2D:DM n=529 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      23

InitMesh: MESH = 80 x 80 x 80 = 512000
InitMesh: Mesh cutoff (required, used) =   150.000   176.881 Ry
New grid distribution [1]: sub = 2

efield: External electric field =    0.200000   -0.200000    0.000000 eV/Ang/e

efield: WARNING!
efield: There is no slab-dipole correction [Slab.DipoleCorrection none] although an external efield is present.
efield: For correct physics Slab.DipoleCorrection should be .true.
efield: This is only for backwards compatibility!


stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -78.111178
siesta: Eions   =       810.513381
siesta: Ena     =       174.034842
siesta: Ekin    =       378.868361
siesta: Enl     =       -70.724082
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -22.099418
siesta: DUscf   =         2.766096
siesta: DUext   =        -0.012320
siesta: Ex      =      -104.466076
siesta: Ec      =       -12.789101
siesta: Exc     =      -117.255177
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -465.635916
siesta: Etot    =      -464.935080
siesta: FreeEng =      -464.935080

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -465.635916     -464.935080     -464.935080  1.164167 -0.718913  8.486901
timer: Routine,Calls,Time,% = IterSCF        1       0.034   3.03
   scf:    2     -466.565701     -465.856332     -465.856332  0.078882 -6.225885  4.255583
   scf:    3     -466.150239     -466.114697     -466.114697  0.066222 -4.995406  0.333276
   scf:    4     -466.100049     -466.116340     -466.116340  0.031217 -4.860601  0.621840
   scf:    5     -466.125973     -466.123714     -466.123714  0.009294 -4.697460  0.034598
   scf:    6     -466.123789     -466.123754     -466.123754  0.000357 -4.697289  0.023691
   scf:    7     -466.123806     -466.123785     -466.123785  0.000370 -4.690489  0.003569
   scf:    8     -466.123786     -466.123786     -466.123786  0.000019 -4.691090  0.002871
   scf:    9     -466.123786     -466.123786     -466.123786  0.000072 -4.693526  0.000092

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000721390
max |H_out - H_in|      (eV) :     0.0000918897
SCF cycle converged after 9 iterations

initwf: No. of electrons corresponding occupied states =            8.000000
initwf: (Total charge - charge in selected states)     =           -0.000000
initwf: Saving wavefunctions in <systemlabel>.TDWF file.

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      23

siesta: E_KS(eV) =             -466.1238

siesta: E_KS - E_eggbox =      -466.1238

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.038961   -0.016118   -0.014980
----------------------------------------
   Max    0.335919
   Res    0.201551    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.335919    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):       -0.12       -0.13       -0.71       -0.00       -0.00       -0.16
(Free)E + p*V (eV/cell)     -465.9240
Target enthalpy (eV/cell)     -466.1238

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -105.447582
siesta: Eions   =       810.513381
siesta: Ena     =       174.034842
siesta: Ekin    =       350.116585
siesta: Enl     =       -63.264488
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -4.868285
siesta: DUscf   =         0.847812
siesta: DUext   =        -0.009543
siesta: Ex      =       -99.896674
siesta: Ec      =       -12.570653
siesta: Exc     =      -112.467327
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -466.123786
siesta: Etot    =      -466.123786
siesta: FreeEng =      -466.123786

siesta: Final energy (eV):
siesta:  Band Struct. =    -105.447582
siesta:       Kinetic =     350.116585
siesta:       Hartree =     453.709924
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =      -0.009543
siesta:         Exch. =     -99.896674
siesta:         Corr. =     -12.570653
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -112.467327
siesta:  Ion-electron =   -1215.580153
siesta:       Ion-ion =      58.106728
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -466.123786
siesta:         Fermi =      -4.693526

siesta: Atomic forces (eV/Ang):
siesta:      1    0.078726    0.335919   -0.006425
siesta:      2   -0.059451   -0.177112   -0.302267
siesta:      3   -0.058236   -0.174925    0.293712
siesta: ----------------------------------------
siesta:    Tot   -0.038961   -0.016118   -0.014980

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.000072   -0.000100   -0.000000
siesta:    -0.000100   -0.000081   -0.000000
siesta:    -0.000000   -0.000000   -0.000446

siesta: Cell volume =       1000.000000 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00000218         -0.00000017  Ry/Bohr**3
siesta:           0.00019981         -0.00001523  eV/Ang**3
siesta:           0.32012492         -0.02439388  kBar
(Free)E+ p_basis*V_orbitals  =        -465.411149
(Free)Eharris+ p_basis*V_orbitals  =        -465.411149

siesta: Electric dipole (a.u.)  =   -0.364790   -0.454960    0.000125
siesta: Electric dipole (Debye) =   -0.927203   -1.156392    0.000319

cite: Please see "TDDFT_h2o.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.490
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.490       1.490   100.00
Setup                  1       0.070       0.070     4.71
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.419       1.419    95.27
geom_init              1       1.026       1.026    68.85
state_init             1       0.107       0.107     7.21
hsparse                1       0.000       0.000     0.00
overlap                1       0.107       0.107     7.19
Setup_H0               1       0.884       0.884    59.32
naefs                  2       0.000       0.000     0.02
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.053       0.107     7.17
nlefsm                 2       0.362       0.724    48.58
DHSCF_Init             1       0.053       0.053     3.59
DHSCF1                 1       0.006       0.006     0.42
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.047       0.047     3.17
REMESH                 1       0.002       0.002     0.15
REORD                 30       0.000       0.012     0.80
PHION                  1       0.020       0.020     1.36
COMM_BSC              47       0.000       0.015     1.01
POISON                12       0.016       0.192    12.89
fft                   24       0.006       0.149     9.98
setup_H               10       0.035       0.352    23.66
DHSCF                 11       0.038       0.423    28.42
DHSCF3                11       0.035       0.389    26.13
rhoofd                11       0.003       0.035     2.35
XC                    11       0.008       0.084     5.67
GXC-CellXC            11       0.006       0.069     4.66
gridxc@cellXC         11       0.006       0.069     4.65
vmat                  11       0.002       0.027     1.78
IterSCF                9       0.036       0.320    21.50
compute_dm             9       0.000       0.001     0.08
diagon                 9       0.000       0.001     0.07
r-eigvec               9       0.000       0.001     0.06
r-buildHS              9       0.000       0.000     0.00
rdiag                 10       0.000       0.001     0.07
rdiag1                10       0.000       0.000     0.00
rdiag2                10       0.000       0.000     0.01
rdiag3                10       0.000       0.000     0.03
rdiag4                10       0.000       0.000     0.01
r-buildD               9       0.000       0.000     0.00
MIXER                  8       0.000       0.000     0.01
initwf                 1       0.000       0.000     0.01
PostSCF                1       0.073       0.073     4.89
DHSCF4                 1       0.034       0.034     2.28
dfscf                  1       0.027       0.027     1.81
overfsm                1       0.000       0.000     0.01
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.01
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:40:43
Job completed
